test/compilable/dtoh_enum.d

```diff
@@ -11,35 +11,35 @@ TEST_OUTPUT:
 #include <stdint.h>
 
 #if !defined(BEGIN_ENUM)
-# define BEGIN_ENUM(name, upper, lower) enum class name {
-# define ENUM_KEY(type, name, value, enumName, upper, lower, abbrev) name = value,
-# define END_ENUM(name, upper, lower) };
+    #define BEGIN_ENUM(name, upper, lower) enum class name {
+    #define ENUM_KEY(type, name, value, enumName, upper, lower, abbrev) name = value,
+    #define END_ENUM(name, upper, lower) };
```

Comment:
According to the C spec, `#` must be at the start of the line. That's why people usually put the spaces after the `#`.

Reply:
Oh ... that's unfortunate.

Reply:
I've never seen any modern c++ compiler rejecting it though, have you?

Reply:
No, but these days it's more that C++ convention follows from that old requirement.
It's weird to indent the `#`, and people will think it looks amateur :P